src/app/public/modules/data-manager/data-manager-toolbar/data-manager-toolbar.component.spec.ts

```diff
@@ -91,6 +97,38 @@ describe('SkyDataManagerToolbarComponent', () => {
   let modalServiceInstance: MockModalService;
   let viewConfig: SkyDataViewConfig;
 
+  function setSearchInput(text: string) {
+    let inputEvent = document.createEvent('Event');
+    let params = {
+      bubbles: false,
+      cancelable: false
+    };
+    inputEvent.initEvent('input', params.bubbles, params.cancelable);
```

Comment:
Would `SkyAppTestUtility.fireDomEvent()` help make this easier?

Reply:
Hmmm. Potentially - I honestly copied this in from another spec. That's a good call though - I'll look at it :)

Reply:
Fixed. I did have to cast the keyboard event but I tried to explain why in a code comment in that spot.